### DISCUSSION TITLE
Node.js 18 / Nuxt 2.16.x support?

### DISCUSSION THREAD

**Discussion by bitfactory-frank-spee** [vuestorefront/shopware-pwa#1988] [General]
Is there any update on Node.js 18 support? And thus a Nuxt 2.16.x update (which is needed for that)?

I see there is still no update on a release (candi)date of Shopware Frontends (which was promised Q1 2023). So for now I think we are stuck with Shopware PWA to build a production version of a shopware headless shop.

Has anyone already tried (and hopefully succeeded) to update the current Shopware PWA version to Nuxt 2.16 / Node.js 18?

**Comment by mkucmus** [comment]
thanks for bringing it up, we need to check first and see what is possible in this matter. Created the issue #1989

**Reply by bitfactory-frank-spee** [reply]
I will add any information, now and in the future, I have for the update there. Most importantly is the Nuxt 2.16 update.
